lib/doekbase/data_api/sequence/assembly/api.py

```diff
@@ -564,7 +534,33 @@ def fetch_contig(start, length):
         return outContigs
 
 
+_as_log = get_logger('AssemblyClientAPI')
+
 class AssemblyClientAPI(AssemblyInterface):
+    def client_method(func):
```

Comment:
Why not define this at a higher scope, so it can be defined only once for all the API classes?